homework/src/index.js

```diff
@@ -33,14 +33,123 @@
   function main(url) {
     fetchJSON(url, (err, data) => {
       const root = document.getElementById('root');
+      const header = createAndAppend('header', root);
+      createAndAppend('p', header, {html: 'HYF Repositories', class: 'header-lable'});
+      const select = createAndAppend('select', header, { id: 'selectMenu' });
       if (err) {
         createAndAppend('div', root, { html: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+        //createAndAppend('pre', root, { html: JSON.stringify(data, null, 2) });
+        /* &&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&& */
+        data.sort((a, b) => a.name.localeCompare(b.name, 'fr', {ignorePunctuation: true}));
```

Comment:
You have probably taken the `localeCompare` arguments from an example in MDN, but there is really reason why should specifically configure it for the French language (argument `'fr'`).

Reply:
Actually i have tired when i was looking for the best and short way to solve it . Finally i copied and pasted it 😆
But after i saw the example